index.js

```diff
@@ -5,14 +5,46 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "pink"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+ console.log('This is what the traveler object looks like: ', traveler)
+let travelerInfo = document.querySelector('.traveler')
+let travelerName = travelerInfo.querySelector('h2')
+let travelerNickname = travelerInfo.querySelector('em')
+let travelerPhoto = travelerInfo.querySelector('img')
+travelerName.textContent = `${traveler.name}`
+travelerNickname.textContent = `${traveler.nickname}`
+travelerPhoto.src = `${traveler.photo}`
+travelerPhoto.alt = "Cairn terrier on wood floor"
```

Comment:
O: !!!! I just looked up images of this breed now and WOW! That looks just like him!! We always suspected he was some sort of terrier mix, but never really new because we adopted him from a pound and they didn't know much about him. So glad you wrote this alt text though; made my evening seeing pics of this breed :D